Documentation/helm-values.rst

```diff
@@ -0,0 +1,1193 @@
+
+.. list-table::
+   :header-rows: 1
+
+   * - Key
+     - Description
+     - Type
+     - Default
+   * - affinity
+     - Pod affinity for cilium-agent.
+     - object
+     - ``{"nodeAffinity":{"requiredDuringSchedulingIgnoredDuringExecution":{"nodeSelectorTerms":[{"matchExpressions":[{"key":"kubernetes.io/os","operator":"In","values":["linux"]}]},{"matchExpressions":[{"key":"beta.kubernetes.io/os","operator":"In","values":["linux"]}]}]}},"podAntiAffinity":{"requiredDuringSchedulingIgnoredDuringExecution":[{"labelSelector":{"matchExpressions":[{"key":"k8s-app","operator":"In","values":["cilium"]}]},"topologyKey":"kubernetes.io/hostname"}]}}``
+   * - agent
+     - Install the cilium agent resources.
+     - bool
+     - ``true``
+   * - alibabacloud.enabled
+     - Enable AlibabaCloud ENI integration
```

Comment:
nit: add period

Garner's recommends that phrases in lists still have punctuation. There are many examples below with missing punctuation, please fix them all.

Reply:
This file is generated from the comments in install/kubernetes/cilium/values.yaml. It's one thing to fix a few typos in there to accommodate spell checking, but I don't think fixing all the punctuation falls under the scope of this PR?

Reply:
Let's do this right, what's the rush? I'll open a PR to fix the values file and once it's merged we can regenerate this file.

Reply:
Hmmm I think the PR is pretty big as is right now. We will need a subsequent PR to address the issue below using, say, https://github.com/norwoodj/helm-docs/pull/99 (and fixing whatever remains to be fixed on the chart), so I'd say we merge this one sooner rather than later — WDYT?

Reply:
I was actually just wondering why we commit this file at all, rather than just generating it when we build the docs?

Reply:
@joestringer Chatting with Bruno we can't think of a good motivation. The only reason that led to its inclusion (and for me not to point it out earlier) is that we have other files similarly generated in the repo already, I'm thinking in particular about the files under Documentation/cmdref. Do you know why we keep them in the repo?

Reply:
@qmonnet @bmcustodio I have a sneaking suspicion that it was just easier because you can build the docs purely with sphinx, you wouldn't need to run other external programs first to generate some of the docs. The same logic would apply here, whether you need to also run these other docker containers and such.

Reply:
Ok so let's keep it in the repo for now? This is something we can revisit in the future, along for the files under Documentation/cmdref, if we want to.